app-ui5-webc/webapp/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+	"_version": "1.12.0",
+	"sap.app": {
+		"id": "sap.ui.demo.todo",
+		"type": "application"
+	},
+	"sap.ui5": {
+		"dependencies": {
+			"minUI5Version": "1.75.0"
+		},
+		"rootView": {
+			"viewName": "sap.ui.demo.todo.view.App",
+			"type": "XML",
+			"async": true,
```

Comment:
This can be removed after adding the "IAsyncContentCreation" to the Component.js